public/contribute/building_crosswalk.md

```diff
@@ -1,282 +1,33 @@
-    libXrender-devel.i686 libXScrnSaver-devel libXtst-devel \
-    libXtst-devel.i686 lighttpd nss-devel nss.i686 pango-devel.i686 \
-    pciutils-devel pciutils-devel.i686 pulseaudio-libs-devel python-pexpect \
-    systemd-devel systemd-devel.i686 xorg-x11-server-Xvfb \
-    xorg-x11-utils zlib-devel.i686 zlib.i686
-    ```
-
-2.  Configure gyp.
-
-    gyp is the build system generator used in Chromium to generate actual build
-    files for Ninja and other systems. It needs a few variables in order to
-    generate build files for Android.
-
-    Have a file called `chromium.gyp_env` in the directory containing your
-    `.gclient` file (so the one above `src/`).
-
-        echo "{ 'GYP_DEFINES': 'OS=android target_arch=ia32', }" > chromium.gyp_env
-
-    If you are building for ARM, use `target_arch=arm` instead of
-    `target_arch=ia32` above.
-
-3.  Configure your setup to generate the Crosswalk projects.
-
-        export GYP_GENERATORS='ninja'
-        python xwalk/gyp_xwalk
-
-4.  To build the contents of the main Crosswalk package for Android, with tools
-    and an embeddable library, you can run:
-
-        ninja -C out/Release xwalk_core_library
-
-    This will create a directory in `out/Release` called `xwalk_core_library`
-    with an architecture-specific Crosswalk library for use when embedding
-    Crosswalk in a project.
-
-    To build Crosswalk's runtime library APK (which can act as the runtime for
-    applications built using Crosswalk's shared mode), run:
-
-        ninja -C out/Release xwalk_runtime_lib_apk
-
-    This will create an APK called `XWalkRuntimeLib.apk` in `out/Release/apks`.
-
-    To build a sample web app APK (for quickly installing/testong on a target),
-    execute:
-
-        ninja -C out/Release xwalk_app_template_apk
-
-    which will create an APK called `XWalkAppTemplate.apk` in
-    `out/Release/apks`.
-
-    Finally, you can also build AAR files for use with Gradle and Maven. The
-    following command will create an architecture-specific AAR file with the
-    main Crosswalk library, as well as an architecture-independent AAR for use
-    with Crosswalk's shared mode:
-
-        ninja -C out/Release xwalk_core_library_aar xwalk_shared_library_aar
-
-    The AAR files will both be located in the `out/Release` directory.
-
-## Running the test suites
-
-If you are interested in running Crosswalk's test suites, you can
-build the test suites from the `src` directory with:
-
-    ninja -C out/Release xwalk_unittest
-    ninja -C out/Release xwalk_browsertest
-
-Then run them with:
-
-    ./out/Release/xwalk_unitttest
-    ./out/Release/xwalk_browsertest
+The Crosswalk Project is structured around and heavily depends on the
+[Chromium project](https://www.chromium.org/Home).
+
+As such, Crosswalk uses the same build system and auxiliary tools as Chromium:
+* `depot_tools` manages source code checkouts, the dependencies among them and
```

Comment:
this didn't show up as a bullet point in my machine. Instead this paragraph was all italic